src/plugins/analysis/file_type/test/test_plugin_file_type.py

```diff
@@ -14,12 +14,9 @@ def setUp(self):
         config = self.init_basic_config()
         self.analysis_plugin = AnalysisPlugin(self, config=config)
 
-    def tearDown(self):
```

Comment:
Are you absolutely sure, that no teardown is needed? What about the db interface and the garbage collection?

Reply:
The function is still called since it exists in the parent class. This just removes the logically empty stub here.

Reply:
you are absolutly right. My fault.